website/docs/extensions/documentation-generation.md

```diff
@@ -85,6 +85,13 @@ fn main() {
 After running `scarb doc`, inside the target directory, you will see the generated documentation in `mdBook` format which consists of:
 
 - The `src` directory, which contains the contents of your book in files with markdown format.
-- The `book.toml` which contains contains settings for describing how to build your book.
+- The `book.toml` which contains settings for describing how to build your book.
 
 Running `scarb doc --output-format json` will result in a single JSON file inside the target directory with collected documentation inside.
+
+Check out the two links below to grasp a better idea on how to work with your documentation:
+
+- [The Cairo Core Libarary Docs](https://docs.cairo-lang.org/core/) to see how a bigger project would look like.
+- [Demo example](https://github.com/starkware-libs/starknet-staking/commit/46e0421a85cf1347aa914262b1df64ca7dc63dd6)
```

Comment:
I am not really sure if we want to share it 🤔 
I feel like it would be best to wait for starknet-staking to use the tool to share their docs officially. 
Would it be a problem if we only left corelib until then? 🤔